embedded-graphics/src/mock_display.rs

```diff
@@ -1,15 +1,14 @@
 //! Mock display for use in tests.
 //!
-//! [`MockDisplay`] can be used to replace a real display in tests. The internal
+//! `MockDisplay` can be used to replace a real display in tests. The internal
```

Comment:
This now needs to be a link again.

Reply:
Good catch. Should've checked the diff...